src/interfaces/IAdaptiveCurveIrm.sol

```diff
@@ -8,24 +8,6 @@ interface IAdaptiveCurveIrm is IIrm {
     /// @notice Address of Morpho.
     function MORPHO() external view returns (address);
 
-    /// @notice Curve steepness (scaled by WAD).
-    /// @dev Verified to be inside the expected range at construction.
-    function CURVE_STEEPNESS() external view returns (int256);
```

Comment:
Do you think that these functions should be accessible outside? Because you can anyway expose, implement and return the direct value from `ConstantLib`